src/store/traction/pacbio/runs/actions.js

```diff
@@ -24,6 +24,8 @@ const buildWell = ({ state }, position) => {
   let generate_hifi = generateHiFiDefault(state.currentRun.system_name)
   let binding_kit_box_barcode = state.currentRun.default_binding_kit_box_barcode || ''
   let [row, column] = splitPosition(position)
+  let loading_target_p1_plus_p2 = state.currentRun.loading_target_p1_plus_p2 || ''
+  let use_adaptive_loading = loading_target_p1_plus_p2 ? 'True' : 'False'
```

Comment:
You could also do the same trick here.  Sorry I didn't point it out before because I thought you may already know.